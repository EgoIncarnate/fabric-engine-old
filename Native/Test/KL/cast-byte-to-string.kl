function entry()
{
  var Byte byte = 17;
  report byte;
  byte = 0;
  report byte;
  byte = 255;
  report byte;
  byte = 4;
  report byte;
}
