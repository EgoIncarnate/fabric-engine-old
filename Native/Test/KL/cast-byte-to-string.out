GlobalList
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        StatementList
          AssignedVarDecl( VarDecl( 'byte', Byte ), ConstSize( 17 ) )
            ConstSize( 17 )
        Report
          Var('byte')
        ExprStatement
          AssignOp(ASSIGN_OP)
            Var('byte')
            ConstSize( 0 )
        Report
          Var('byte')
        ExprStatement
          AssignOp(ASSIGN_OP)
            Var('byte')
            ConstSize( 255 )
        Report
          Var('byte')
        ExprStatement
          AssignOp(ASSIGN_OP)
            Var('byte')
            ConstSize( 4 )
        Report
          Var('byte')
17
0
255
4
